### PR TITLE
bug 1152386 - Adding tests for new, edit, and translate within the wiki

### PR DESCRIPTION
**Pull request by darkwing**
Adds testing for edit and translate screens, ensuring CKEditor loads

### CODE REVIEW

**Comment by darkwing** [comment]
Merged the other PRs into this.

File: tests/ui/_cli.js [hunk 1]
```diff
@@ -30,16 +30,22 @@ define({
...
         if(args.wd == undefined) {
             greps.push('requires-doc');
+            console.log('No wiki document (-wd) provided.  Most wiki tests will be skipped.');
         }
 
         // Set the final GREP value
         args.grep = greps.length ? ('^(?!.*?\\[' + greps.join('|') + '\\])') : '';
 
+        if(args.grep) {
+            console.log('Command line arguments have forced a grep to skip tests:  ' + args.grep);
```
Comment (openjck) [inline tests/ui/_cli.js]: Hm, I'm not sure I understand. Is there another way of phrasing this? What will be different from the point of view of the person running the test?

Comment (darkwing) [inline tests/ui/_cli.js]: It's basically summarizing the two comments above.

Comment (openjck) [inline tests/ui/_cli.js]: Ah, so it will appear only when another error message also appears. Okay, that makes sense then.

**Pull request merged**